Navigation/Profile/PostTableViewCell.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  PostTableViewCell.swift
+//  Navigation
+//
+//  Created by Irina on 07.02.2022.
+//
+
+import UIKit
+
+class PostTableViewCell: UITableViewCell {
+    var post: Post? {
+        didSet {
+            guard let post = post else { return }
+            authorLabel.text = post.author
+            postImageView.image = post.image
+            descriptionLabel.text = post.description
+            
+            let likes = post.likes
+            let likesString = "Likes: \(String(likes))"
+            likesLabel.text = likesString
+            
+            let views = post.views
+            let viewsString = "Views: \(String(views))"
+            viewsLabel.text = viewsString
+        }
+    }
+
+    private let authorLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 20, weight: .bold)
+        label.textColor = .black
+        label.numberOfLines = 2
+        return label
+    }()
+    
+    private let postImageView: UIImageView = {
+        let imageView = UIImageView()
+        /// Adapts to screen size.
+        imageView.contentMode = .scaleAspectFit
```

Comment:
Actually it adapts image to postImageView size

Reply:
Thanks!